hlxsites/merative2
# feat(merative-history): add static build artifacts

## Issue

Fixes https://jira.wh-sdlc.watson-health.ibm.com/browse/MERATIVE-726

## Description

**Changed**

- This PR adds the static build artifacts (html/css/js) of the Merative's History page (https://www.ibm.com/resources/watson-health/merative-history/) that's currently hosted on ibm.com.
- New directory `/resources/history/`
- We need a way to host these files because IBM will most likely sunset this URL on **June 30, 2023**. 

**Note**
- We need to figure out a way to set `index.html` as the home of `/resources/history/`


## Design Specs

- Figma Link - N/A

## Test URLs
  
![Screen Shot 2023-06-22 at 4 33 49 PM](https://github.com/hlxsites/merative2/assets/1815714/06360cf1-3b67-40f9-a049-40473a345acb)

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.live/
- After (Changes from this PR): https://feat-add-merative-history--merative2--proeung.hlx.page/resources/history/index.html
  
## Testing Instruction

- Open the IBM-hosted version of Merative's history - https://www.ibm.com/resources/watson-health/merative-history/
- Compare it with the version in this repo - https://feat-add-merative-history--merative2--proeung.hlx.page/resources/history/index.html
- Ensure that the design and interaction is the same.
